com.rustdesk.RustDesk.json

```diff
@@ -81,9 +84,12 @@
           "sha256": "f7d675ed2160c05b2206b6b7e50a2d369b6ed7f63abe0e89b9f38f9a29793fb4",
           "dest-filename": "rustdesk.deb",
           "x-checker-data": {
-            "type": "anitya",
-            "project-id": "309593",
-            "url-template": "https://github.com/rustdesk/rustdesk/releases/download/$version/rustdesk-$version-x86_64.deb"
+            "type": "json",
+            "url": "https://api.github.com/repos/rustdesk/rustdesk/releases/latest",
+            "version-query": ".tag_name",
+            "timestamp-query": ".published_at",
+            "url-query": ".assets[] | select(.name == \"rustdesk-\\($version)-x86_64.deb\").browser_download_url",
+            "is-main-source": true
```

Comment:
@guihkx hello. I don't see any information [in the documentation](https://github.com/flathub-infra/flatpak-external-data-checker?tab=readme-ov-file#json-checker) about the `"is-main-source"` property. Could you please provide a link where I can find information about it to better understand what is happening here? Thank you in advance!

Reply:
Oof, that is poorly documented... But basically, `is-main-source: true` tells the Flathub bot to automatically add a new `<release>` tag to the [`*.metainfo.xml`](https://github.com/flathub/com.rustdesk.RustDesk/blob/8731340a620274d048eedd2384954e6f3baef850/com.rustdesk.RustDesk.metainfo.xml#L59) file.

That is only necessary when we have multiple `source` entries in the Flatpak manifest file, so that the bot knows which one is the app.

Reply:
@guihkx I'm not sure about adding it then. In my opinion, it's better not to add properties that are undocumented or experimental. Also, does this property only need to be here and not in the "aarch" source?

Also, as I [see in the documentation](https://github.com/flathub-infra/flatpak-external-data-checker?tab=readme-ov-file#motivation), the "x-checker-data" is essential for those configurations that have the "extra-data" source. It's probably better to remove "x-checker-data" from the manifest completely, what do you think?

Reply:
@guihkx I'm not sure about adding it then. In my opinion, it's better not to add properties that are undocumented or experimental. Also, does this property only need to be here and not in the "aarch" source?

Also, as I [see in the documentation](https://github.com/flathub-infra/flatpak-external-data-checker?tab=readme-ov-file#motivation), the "x-checker-data" is essential for those configurations that have the "extra-data" source. It's probably better to remove "x-checker-data" from the manifest completely, what do you think?

Reply:
>So do I understand correctly that this property only needs to be here and not in the "aarch" source? 

I think it's redundant to have it on both sources, though I don't think it would cause side effects if we did that. But I do know for sure that having it on only one of them is enough.

> the "x-checker-data" is essential for those configurations that have the "extra-data" source.

Yes, it's essential for `extra-data` sources, but it doesn't mean it's not useful for "regular" sources, too.

>So, it's probably better to remove "x-checker-data" from the manifest completely, what do you think?

I don't agree, but we could do that. The problem is that upgrading to new RustDesk versions would have to be a manual labor. And trust me, that labor gets old very quick.